### PR TITLE
Callset statistics [VS-560]

### PR DESCRIPTION
**Pull request by mcovarr**
Successful Quickstart run [here](https://app.terra.bio/#workspaces/gvs-dev/mlc%20GVS%20Quickstart%20v3/job_history/160b0035-c84e-4e54-a343-0e9c570c1da1), has not yet been run on larger datasets.

### CODE REVIEW

**Review by gbggrant — COMMENTED** [review]
Where do the outputs go? I don't see anything in the ~{extract_prefix}_statistics table that got created in my [test run](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20WGS%2010K%20Callset/job_history/889f5c2e-cbb7-482c-a49a-d923aebd8207). 

Also, I kind of think a text file as output would be very useful for analysis/reporting.

File: scripts/variantstore/wdl/GvsCallsetStatistics.wdl [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+        bq --project_id=~{project_id} show ~{dataset_name}.~{statistics_table}
+        BQ_SHOW_RC=$?
+        set -o errexit
+
+        if [ $BQ_SHOW_RC -eq 0 ]; then
+            echo "Output statistics table already exists, exiting."
+            exit 1
+        fi
+
+        # Schemas extracted programatically: https://stackoverflow.com/a/66987934
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCallsetStatistics.wdl:102]: cool!

File: scripts/variantstore/wdl/GvsCallsetStatistics.wdl [hunk 2]
```diff
@@ -0,0 +1,707 @@
...
+            ti_count,
+            tv_count,
+            snp_het_count,
+            snp_homvar_count,
+            indel_het_count,
+            indel_homvar_count,
+            singleton,
+            pass_qc
+        )
+        SELECT "~{filter_set_name}" filter_set_name,
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCallsetStatistics.wdl:467]: I know that none of the explanations are in the code that you are looking at to write this wdl, but I think getting Lee to add some context about what is being calculated would be really helpful. I'm fine with that being a future ticket

**Comment by gbggrant** [comment]
So, ran it on a previously run set and it [failed](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20WGS%2010K%20Callset/job_history/3ec41d9c-046b-4008-9ee6-b2f3a07aec45) as there were rows in the database tables.
Then dropped the tables and it [failed](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20WGS%2010K%20Callset/job_history/ce35b598-c634-48fb-9466-a98cfe7b6070) because I dropped the tables. Presumably need a `volatile=true` on the CreateTables task (and make it find or create)?

**Review by gbggrant — APPROVED** [review]

**Review by rsasch — CHANGES_REQUESTED** [review]
As far as I can tell, this workflow creates the statistics table but does not output the contents into a TSV or CSV, which is what we deliver along with the callset.  Would it be possible to add an export to TSV to a specified GCS location to the CollectStatistics task (or a new one)?

**Comment by rsasch** [comment]
On the good news front, I compared an export of the "statistics_table" to the callset stats file I generated for Beta and they matched! 👍🏻

**Comment by mcovarr** [comment]
now with export to CSV

**Comment by mcovarr** [comment]
Successful run here https://app.terra.bio/#workspaces/gvs-dev/mlc%20GVS%20Quickstart%20v3/job_history/3cacea14-3c61-42ef-9370-2677af5e14d3

**Review by rsasch — APPROVED** [review]

**Pull request merged**